### PR TITLE
Autocomplete

### PR DESCRIPTION
**Pull request by ChrisChrisLoLo**
Add an autocomplete component. Clicking on a result will redirect you to the page of that author. Displays display name and host.

Screenshot_20210405_152844

### CODE REVIEW

**Comment by ChrisChrisLoLo** [comment]
As a heads up, I think the package.json got misconfigured at some point. The changes made were generated via `npm install`. This hiccup occurred with Monica as well. I think we should commit this newly generated package.json.lock file

File: front-end/src/components/ProfilePic.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  style?: React.CSSProperties
+}
+
+export default function profilePic(props:Props){
+  if (props.author.github?.includes("github.com")) {
+    return <img width="100%" src={props.author.github + ".png"} alt="Card image cap" style={props.style}/>
+  } else {
+    return <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor" style={props.style}>
+              <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M16 7a4 4 0 11-8 0 4 4 0 018 0zM12 14a7 7 0 00-7 7h14a7 7 0 00-7-7z" />
+            </svg>
```
Comment (raysarinas) [inline front-end/src/components/ProfilePic.tsx]: could we move this to `assets/icons.jsx`? my PR reverts the changes I made to use the `react-icons` package (re the bloating stuff you mentioned)

**Review by raysarinas — APPROVED** [review]
LGTM, just one comment

File: front-end/src/components/ProfilePic.tsx (hunk 1 above)
Comment (ChrisChrisLoLo) [reply front-end/src/components/ProfilePic.tsx]: Good catch! I've implemented this change

**Pull request merged**